### PR TITLE
add checks for includePaths for sass-loader

### PR DESCRIPTION
**Pull request by DisruptiveMind**
### Description
Original issue opened at #1968 
This PR adds checks to support `includePaths` for the following plugins:

- `gatsby-plugin-sass`
- `gatsby-plugin-postcss-sass`

### Tests
Several tests have been added for both plugins to account for the various combinations.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
     "eslint-plugin-prettier": "^2.2.0",
     "eslint-plugin-react": "^7.3.0",
     "flow-bin": "^0.42.0",
+    "gatsby-1-config-css-modules": "^1.0.3",
```
Comment (KyleAMathews) [inline package.json]: This is what was missing to make the tests work https://github.com/gatsbyjs/gatsby/pull/1964/commits/d09cc11d3c5e29bb28ee9cff7807e95cb222f463 so you can remove this.

**Comment by KyleAMathews** [comment]
Deploy preview failed.

Built with commit 5def0072c1d0a105c87440e3b5933c473150a1d2

https://app.netlify.com/sites/using-glamor/deploys/59a700bb424ef26ad4800b9d

**Comment by jquense** [comment]
I think we could avoid doing this for each new option by making the query just: 
```
`sass-loader?${JSON.stringify(options)`
```

generally loaders all accept json strings as well

**Comment by DisruptiveMind** [comment]
@jquense that's a better approach!

**Comment by DisruptiveMind** [comment]
@jquense at a quick run tests fail using your suggestion. I'll take a look later in depth.

**Comment by KyleAMathews** [comment]
Is this PR ready to go?

**Comment by DisruptiveMind** [comment]
@KyleAMathews looks ready to me. The netlify deploy failed though, is there concern there?

**Comment by KyleAMathews** [comment]
I merged master so that should fix things.

**Comment by KyleAMathews** [comment]
Thanks!

**Comment by jlengstorf** [comment]
Hiya @DisruptiveMind! 👋

This is definitely late, but on behalf of the entire Gatsby community, I wanted to say thank you for being here.

Gatsby is built by awesome people like you. Let us say “thanks” in two ways:

1.  **We’d like to send you some Gatsby swag.** As a token of our appreciation, you can go to the [Gatsby Swag Store][store] and log in with your GitHub account to get a coupon code good for one free piece of swag. (We’ve got t-shirts and hats, plus some socks that are really razzing our berries right now.)
2.  **If you’re not already part of it, we just invited you to join the Gatsby organization on GitHub.** This will add you to our team of maintainers. You’ll receive an email shortly asking you to confirm. By joining the team, you’ll be able to label issues, review pull requests, and merge approved pull requests.

If you have questions, please don’t hesitate to reach out to us: tweet at [@gatsbyjs][twitter] and we’ll come a-runnin’.

Thanks again! 💪💜

[store]: https://store.gatsbyjs.org
[twitter]: https://twitter.com/gatsbyjs

**Pull request merged**